.github/workflows/build-test.yml

```diff
@@ -41,7 +41,8 @@ jobs:
         with:
           STORE_NX_CACHE: 'true'
       - id: test
-        run: pnpm nx affected --target test --base=remotes/origin/main
+        # Test runners run in parallel, so we use `--parallel=1` to avoid running them all at once
+        run: pnpm nx affected --target test --base=remotes/origin/main --parallel=1
```

Comment:
These were starting to clobber together, leading to test timeouts. The test runners already run individual test files in parallel, so there's not much benefit to running the test scripts in parallel too.